### ISSUE TITLE
hide-pagination-control does not work

### ISSUE THREAD

**Issue by kornfein** [PredixDev/px-data-table#40]
## Expected behavior and actual behavior:
adding the hide-pagination-control property should term off pagination. Our px-data-table has a small fixed number of rows we do not want pagination.

## Steps to reproduce the problem:
       <px-card id="tableCard" header-text="Hours Details for Work Order {{jobName}}">
            <px-data-table id="dataTable" hide-pagination-control table-data="{{data}}">
                <px-data-table-column name="hoursInBuild" label="Total Hours in Build"></px-data-table-column>
                <px-data-table-column name="currentHours" label="Current Hours"></px-data-table-column>
                <px-data-table-column name="stdHoursRemaining" label="Hours remaining"></px-data-table-column>
                <px-data-table-column name="dueDate" label="Due"></px-data-table-column>
                <px-data-table-column name="estimatedHoursToCompletion" label="Estimation of completion (in hours)"></px-data-table-column>
                <px-pagination style="display: none;"></px-pagination>
            </px-data-table>
        </px-card>

## Environment (_component version number, Browser (including version), operating system, hardware, etc_):
Chrome

## Screenshots (_optional, but very helpful_):
![image](https://cloud.githubusercontent.com/assets/5863663/15974661/99b1bf6c-2f17-11e6-9b3d-010233e48d82.png)

**Comment by alaviamir** [comment]
@kornfein: What version are you currently using? I believe it was introduced in 1.1.0
Can you try upgrading the version of px datatable?

**Comment by tuxji** [comment]
Mark had to leave early so I took over.  We had used predix-seed v1.2.16 as our app's base, so we were using px-data-table v0.3.0.  I applied the changes between predix-seed v1.2.16 and v1.4.0 to bower.json and package.json and ran npm install and bower install --force-latest (needed that option to fix bower ECONFLICT Unable to find suitable version for polymer-font-awesome).  That upgraded px-data-table to v1.8.0, so hide-pagination-control works for us now.
